src/styles/blocks/header.scss

```diff
@@ -0,0 +1,21 @@
+.header {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-position: 40% 10%;
+  background-size: cover;
+  padding: 2.5rem 1.5rem;
+  background-image: url("../images/girl2.png");
```

Comment:
header image of a woman looks well aligned in Desktop and mobile mode but needs to align in IPad mode